### ISSUE TITLE
Make all Razor Nexus DDRIT tests be non-experimental

### ISSUE THREAD

**Issue by ryanbrandenburg** [dotnet/aspnetcore#26640]
Continuation of https://github.com/dotnet/aspnetcore/issues/23964. Now that we have SOME tests Required we should drive our remaining tests to a reliability level where they can be required as well.

**Comment by ryanbrandenburg** [comment]
Let's also ensure that the Diagnostics tests test pull diagnostics (@TanayParikh).

**Comment by TanayParikh** [comment]
> Let's also ensure that the Diagnostics tests test pull diagnostics (@TanayParikh).

Yep, they'll test the "default" diagnostics, aka. pull diagnostics in codespaces once we switchover. The DDRIT test shouldn't require any changes as it's just looking for the squiggles to appear.

**Comment by NTaylorMullen** [comment]
This is getting usurped by: https://github.com/dotnet/aspnetcore/issues/26831

Moving this to backlog
